firefox-ios/Client/Coordinators/CredentialAutofillCoordinator.swift

```diff
@@ -36,6 +37,10 @@ class CredentialAutofillCoordinator: BaseCoordinator {
 
     // MARK: - Methods
 
+    private func currentTheme() -> Theme {
+        return themeManager.currentTheme(for: windowUUID)
+    }
```

Comment:
see this private func created, but don't see it being used below

Reply:
Fixed, ty; this was meant to be used in place of `themeManager.currentTheme(for: windowUUID)`. Code is equivalent but I was trying to move towards this pattern since I thought it was slightly cleaner.